File: sample_programs/python_sample_programs/1cellumts.py
from __future__ import print_function
import time
import unwired
from unwired.rest import ApiException
from pprint import pprint

# create an instance of the API class
api_instance = unwired.GEOLOCATIONApi()

#cell schema
c1=unwired.CellSchema(lac=38996,cid=40907278)
celldata=[c1]

#Geolocation Schema
geolocation_schema = unwired.GeolocationSchema(token="YOUR_API_KEY",radio="cdma",mcc=310,mnc=404,cells=celldata,address=1)

try:
    # Geolocation
    api_response = api_instance.geolocation(geolocation_schema)
    pprint(api_response)
except ApiException as e:
    print("Exception when calling GEOLOCATIONApi->geolocation: %s\n" % e)

File: sample_programs/java_sample_programs/1cellumts.java
//import unwired.ApiException;
//import com.unwired.client.api.GeolocationApi;


public class GeolocationApiExample {

    public static void main(String[] args) {
        
     GeolocationApi apiInstance = new GeolocationApi();
    GeolocationSchema geolocationSchema = new GeolocationSchema(); // GeolocationSchema | 
    //Cell Schema
    CellSchema c1=new CellSchema();
    c1.lac(38996);
    c1.cid(40907278);

    List celldata = new ArrayList();
    celldata.add(c1);


    geolocationSchema.token("YOUR_API_TOKEN");
    geolocationSchema.radio("cdma");
    geolocationSchema.mcc(310);
    geolocationSchema.mnc(404);
    geolocationSchema.cells(celldata);
    geolocationSchema.address(AddressSchema.NUMBER_1);

        try {
            Object result = apiInstance.geolocation(geolocationSchema);
            System.out.println(result);
        } catch (ApiException e) {
            System.err.println("Exception when calling GeolocationApi#geolocation");
            e.printStackTrace();
        }
    }
}

